packages/remix-entry-server/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@vercel/remix-entry-server",
+  "version": "0.0.1",
+  "description": "Isomorphic `entry.server` implementation for Vercel's Serverless and Edge runtimes",
+  "homepage": "https://vercel.com/docs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/vercel.git",
+    "directory": "packages/remix-entry-server"
+  },
+  "main": "./dist/entry.server.node.js",
+  "browser": "./dist/entry.server.edge.js",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@remix-run/node": "1.12.0",
+    "isbot": "3.6.5",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@types/react-dom": "18.0.10",
```

Comment:
For safety, it's probably a good idea to add `execa` and `fs-extra` to the `devDependencies`.